src/code/components/modal-alert.js

```diff
@@ -86,21 +87,32 @@ class ModalAlert extends React.Component {
     if (this.props.explanation) {
       explanationView = <p>{t(this.props.explanation)}</p>;
     }
-    return (
-      <Modal  aria-labelledby='modal-label'
-              style={modalStyle}
-              backdropStyle={backdropStyle}
-              show={this.props.show}
-              onHide={this.props.onHide} >
-        <div style={dialogStyle(this.props.top)} >
-          <h4 id='modal-label'>{t(this.props.message)}</h4>
-          {awardView}
-          {explanationView}
-          {leftButton} {rightButton}
-        </div>
-      </Modal>
-    );
+    if (this.props.show) {
+      // Only trial-end and next-trial modals are shown
```

Comment:
I'm still using `modal-alerts` to show information that don't belong in the HUD scroll. For now, that just includes the "next level" button and the end of level feedback